frontend/src/pages/projects/utils.ts

```diff
@@ -21,3 +21,6 @@ export const getCustomNotebookSize = (
     requests: {},
   },
 });
+
+export const getPvcAccessMode = (pvc: PersistentVolumeClaimKind | undefined): string =>
+  pvc?.spec.accessModes[0] ?? '';
```

Comment:
I would rather keep this helper pure instead of having it handle undefines or returning an empty string. Do that logic where it is needed in the usage of this func

```suggestion
export const getPvcAccessMode = (pvc: PersistentVolumeClaimKind): string | undefined =>
  pvc.spec.accessModes[0];
```